app/services/domain/expenses/car-expense.js

```diff
@@ -0,0 +1,14 @@
+const BaseExpense = require('../../../services/domain/expenses/base-expense')
+const EXPENSE_TYPE = require('../../../constants/expense-type-enum')
+
+class CarExpense extends BaseExpense {
+  constructor (claimId, from, to) {
+    super(claimId, EXPENSE_TYPE.CAR, null, null, from, to, null, null, null)
+  }
+
+  isValid () {
+    // TODO: Implement validation.
+  }
+}
```

Comment:
as above, need validation errors for toll/parking throw in here, could use `get` property to retrieve tollExpense object if toll was checked